vendor/bat-native-ads/src/bat/ads/internal/account/confirmations/confirmations_state.cc

```diff
@@ -364,6 +364,10 @@ bool ConfirmationsState::GetFailedConfirmationsFromDictionary(
         confirmation_dictionary->FindStringKey("ad_type");
     if (ad_type) {
       confirmation.ad_type = AdType(*ad_type);
+    } else {
+      // Migrate legacy confirmations, this value is not used right now so safe
+      // to set to |kAdNotification|
+      confirmation.ad_type = AdType::kAdNotification;
```

Comment:
Value must be anything other than `kUndefined` and is only used for display purposes so safe to set to `kAdNotification` for legacy confirmations